capella2polarion/connectors/polarion_worker.py

```diff
@@ -265,7 +297,9 @@ def compare_and_update_work_item(
                     new.type,
                     new.title,
                 )
-                self.client.delete_work_item_links(list(delete_links.values()))
+                self.project_client.work_items.links.delete(
+                    list(delete_links.values())
+                )
```

Comment:
I don't know how much time we could gain here to spare this type conversion... I'd expect to just pass an iterable here instead of constructing a new list out of the `delete_links.values()`. I created an [issue in the REST API client library](https://github.com/DSD-DBS/polarion-rest-api-client/issues/49) for it. We can improve this in another PR, when ever we find time to refactor (see #107).

Reply:
so this one can be resolved for now?